channels-rke2.yaml

```diff
@@ -969,3 +969,11 @@ releases:
         repo: rancher-rke2-charts
         version: v3.9-build2022102805
     featureVersions: *featureVersions-v1
+  - version: v1.24.9-rc1+rke2r2
```

Comment:
I think we should replace v1.24.9+rke2r1 with v1.24.9-rc1+rke2r2. R1 never shipped in a Rancher release as far as I know, and we don't want anyone to be able to pick it.